lua/example_of_dynamic_config_request.lua

```diff
@@ -0,0 +1,8 @@
+-- TODO Add examples for other operations. For example:
+-- Functions to add/del/list the GK FIB entries.
+-- Functions to list the ARP table.
+-- Functions to list the ND table.
+-- Functions to process the GT policies.
+-- ......
+
+return dy_default_proc()
```

Comment:
Instead of bringing `dy_default_proc()` to this file, just replace `dy_default_proc()` with the string it returns.